crates/lib/src/qpu/execution.rs

```diff
@@ -64,7 +64,7 @@ pub(crate) enum Error {
     #[error("Program when getting substitutions for program: {0}")]
     Substitution(String),
     #[error("Problem making a request to the QPU: {0}")]
-    QpuApiError(#[from] super::api::QpuApiError),
+    QpuApi(#[from] super::api::QpuApiError),
```

Comment:
These variant names were failing clippy's [enum_variant_name](https://rust-lang.github.io/rust-clippy/master/index.html#/enum_variant) lint. Since the enum is crate private, I went ahead and renamed them.